DTS-STN/canadian-dental-care-plan
# Add children information route for adult child flow

### Description


Add children information route for adult child flow.

### Related Azure Boards Work Items


### Screenshots (if applicable)


### Checklist


### Test Instructions


Go throught adult child flow. Add new child from children HUB page. You should land on the children information page and be able to save. Of course the child is still in new state since it misses the dental benefits and insurance.

### Additional Notes
